core/src/test/scala/org/apache/spark/status/AppStatusStoreSuite.scala

```diff
@@ -95,10 +123,18 @@ class AppStatusStoreSuite extends SparkFunSuite {
 
   private def newTaskData(i: Int): TaskDataWrapper = {
     new TaskDataWrapper(
-      i, i, i, i, i, i, i.toString, i.toString, i.toString, i.toString, false, Nil, None,
+      i, i, i, i, i, i, i.toString, i.toString, "SUCCESS", i.toString, false, Nil, None,
       i, i, i, i, i, i, i, i, i, i,
       i, i, i, i, i, i, i, i, i, i,
       i, i, i, i, stageId, attemptId)
   }
 
+  private def failedTaskData(i: Int): TaskDataWrapper = {
```

Comment:
better to have the status be a parameter to the existing method (with a default value)

Reply:
Thanks. done.